### ISSUE TITLE
Dashboard parameter dropdown does not apply background color theming in the embedding sdk

### ISSUE THREAD

**Issue by heypoom** [metabase/metabase#46572]
When using the following theme options, the dashboard parameter dropdown does not take into account the background color (`colors.background`) - so it's taking a white background color instead of the themed dark color.

```ts
    colors: {
      brand: '#509EE3',
      filter: '#7172AD',
      border: '#5A5F6B',
      background: '#2D353A',
      'background-hover': '#2D353A',
      positive: '#84BB4C',
      negative: '#ED6E6E',
    },
```

![Image](https://github.com/user-attachments/assets/33d51139-0fbc-40a3-8c6a-2b0e0f84ee1a)

PS. Looks like the text color is being followed correctly, but because the background is not set correctly, it looks broken:

```ts
      'text-primary': '#FFFFFF',
      'text-secondary': '#FFFFFF',
      'text-tertiary': '#FFFFFF',
```

![Image](https://github.com/user-attachments/assets/ca694901-0b4d-4aed-905d-765d5cc5d514)

**Comment by heypoom** [comment]
Already fixed as of [SDK v0.1.33](https://www.npmjs.com/package/@metabase/embedding-sdk-react).

**Comment by heypoom** [comment]
![Image](https://github.com/user-attachments/assets/8a242152-c682-41cd-b673-fb3103efaaa1)
